### PR TITLE
feature/header: Add header  

### PR DESCRIPTION
**Pull request by juan-ortizc**
1. Here the link to the task in trello: https://trello.com/c/v4tn8CSR/11-header-3
2. Here the task's design in zepplin: https://app.zeplin.io/project/5f15bfe784a1689e60712353/screen/5f15c021495aa48e6122c150
3. Here a screenshot of the task resolved
image

4. Create component header
5. Create helpers folder 
6. Create function that creates html tags with it's respective class. This function was allocated within the helpers folder

### CODE REVIEW

**Review by ArleyAV — APPROVED** [review]
Well done Juanda

**Review by omeneses — COMMENTED** [review]
The PR does not meet requirements outlined at https://docs.google.com/document/d/1WM3jr92W3KV4uvECwLxMvegHBvdyxsTSWIGtFMjyQNI/edit
section: PR's structure
(Missing links to task on Trello and Zeplin's design)

**Comment by juan-ortizc** [comment]
@omeneses 

> The PR does not meet requirements outlined at https://docs.google.com/document/d/1WM3jr92W3KV4uvECwLxMvegHBvdyxsTSWIGtFMjyQNI/edit
> section: PR's structure
> (Missing links to task on Trello and Zeplin's design)


 I already did the changes to PRs that you suggested

File: src/js/components/header.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { createElement } from "../helpers/helpers.js";
```
Comment (manuugm) [inline src/js/components/header.js]: The .js in this line is not necessary

File: src/js/index.js [hunk 2]
```diff
@@ -1,26 +1,28 @@
 // Component imports
-import { defaultComp } from './components/default';
+import { defaultComp } from "./components/default";
```
Comment (manuugm) [inline src/js/index.js]: For js files, it's recommended to use single quotes instead of double-quotes. Please replace this where needed

File: src/js/index.js [hunk 3]
```diff
@@ -1,26 +1,28 @@
...
 const components = {
   default: defaultComp,
+  header: headerSection,
 };
 
 const getCurrentPage = () => {
-  const path = window.location.href.split('/');
-  const location = path.slice(-1)[0] || 'home';
-  return location.replace('.html', '');
+  const path = window.location.href.split("/");
```
Comment (manuugm) [inline src/js/index.js]: Same here!!

File: src/scss/components/header.scss [hunk 4]
```diff
@@ -0,0 +1,14 @@
+.header {
+  box-shadow: 0 2px 4px 0 rgba($black, 0.06);
+  &__container {
```
Comment (manuugm) [inline src/scss/components/header.scss:4]: Please put a line break between class blocks

File: src/scss/components/header.scss [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    padding: 16px 20px;
+  }
+  &__link {
+    display: block;
+    height: auto;
+    margin: 0 auto;
+    width: 154px;
+  }
+  &__logo {
```
Comment (manuugm) [inline src/scss/components/header.scss]: Please remove empty class

File: src/scss/main.scss [hunk 6]
```diff
@@ -1,10 +1,13 @@
 @charset 'UTF-8';
 
 // 1. Configuration and helpers
-@import 'abstracts/variables';
-@import 'abstracts/functions';
-@import 'abstracts/mixins';
+@import "abstracts/variables";
```
Comment (manuugm) [inline src/scss/main.scss]: Change double quote for a single quote

File: src/js/index.js (hunk 2 above)
Comment (juan-ortizc) [reply src/js/index.js]: Resolved

File: src/js/components/header.js (hunk 1 above)
Comment (juan-ortizc) [reply src/js/components/header.js]: Resolved

File: src/js/index.js (hunk 3 above)
Comment (juan-ortizc) [reply src/js/index.js]: Resolved

File: src/scss/components/header.scss (hunk 4 above)
Comment (juan-ortizc) [reply src/scss/components/header.scss:4]: Resolved

File: src/scss/components/header.scss (hunk 5 above)
Comment (juan-ortizc) [reply src/scss/components/header.scss]: Resolved

File: src/scss/main.scss (hunk 6 above)
Comment (juan-ortizc) [reply src/scss/main.scss]: Resolved

File: src/about.html [hunk 7]
```diff
@@ -1,13 +1,16 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang='en'>
```
Comment (jsebax) [inline src/about.html]: @juan-ortizc single-quotes are only used in JS files, for HTML files use double-quotes. Please revert the changes on this file

File: src/data/data.js [hunk 8]
```diff
@@ -22,56 +32,59 @@ const pageData = {
...
-        title: 'Como recoger el aceite'
+        title: 'Como recoger el aceite',
       },
       {
         name: 'timelineItem',
         index: 0,
         title: 'Deja enfriar el aceite',
         text: 'Con esto evitamos posibles quemaduras o derretir el embase.',
-        icon: ''
+        icon: ',
```
Comment (jsebax) [inline src/data/data.js]: Don't remove the closing quote on this property, otherwise it will cause an error. Leave the icon value as it was before

File: src/data/data.js [hunk 9]
```diff
@@ -22,56 +32,59 @@ const pageData = {
...
       },
       {
         name: 'timelineItem',
         index: 1,
         title: 'Filtra el aceite',
         text:
           'Filtre o cuele el aceite para que no pasen restos de comida, y viértelo en un envase de plástico, que puede ser el mismo frasco donde viene el aceite, cierre bien con la tapa.',
-        icon: ''
+        icon: ',
```
Comment (jsebax) [inline src/data/data.js]: Same here

File: src/data/data.js [hunk 10]
```diff
@@ -22,56 +32,59 @@ const pageData = {
...
       },
       {
         name: 'warningComp',
         title: 'Evita mezclar el aceite usado con agua.',
-        icon: ''
+        icon: ',
```
Comment (jsebax) [inline src/data/data.js]: Same here

File: src/data/data.js [hunk 11]
```diff
@@ -22,56 +32,59 @@ const pageData = {
...
       },
       {
         name: 'timelineItem',
         index: 2,
         title: 'Trae el aceite',
-        icon: ''
+        icon: ',
```
Comment (jsebax) [inline src/data/data.js]: Same here

File: src/data/data.js [hunk 12]
```diff
@@ -80,168 +93,170 @@ const pageData = {
...
+          'Contaminando ríos y depósitos de agua, creando focos de hongos y bacterías.',
+        image: ',
       },
       {
         name: 'percentTimelineItem',
         percent: '20%',
         title: 'Lo tiran a la basura',
         text: 'Filtrándose en suelos, poniendo en riesgo la vida al rededor.',
-        image: ''
+        image: ',
```
Comment (jsebax) [inline src/data/data.js]: Same as before. Don't remove the closing quote. Please check other places where the closing quote is missing, and fix it.

File: src/about.html (hunk 7 above)
Comment (juan-ortizc) [reply src/about.html]: resolved

File: src/data/data.js (hunk 8 above)
Comment (juan-ortizc) [reply src/data/data.js]: resolved

File: src/data/data.js (hunk 9 above)
Comment (juan-ortizc) [reply src/data/data.js]: resolved

File: src/data/data.js (hunk 10 above)
Comment (juan-ortizc) [reply src/data/data.js]: resolved

File: src/data/data.js (hunk 11 above)
Comment (juan-ortizc) [reply src/data/data.js]: resolved

File: src/data/data.js (hunk 12 above)
Comment (juan-ortizc) [reply src/data/data.js]: resolved

File: src/about.html [hunk 13]
```diff
@@ -1,13 +1,16 @@
...
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>SRAUC</title>
   <link rel="stylesheet" href="./scss/main.scss">
 </head>
+
 <body>
   <div id="root"></div>
   <script src="./js/index.js"></script>
 </body>
+
```
Comment (camigomezdev) [inline src/about.html]: Please don't send changes in this file

File: src/js/helpers/helpers.js [hunk 14]
```diff
@@ -0,0 +1,7 @@
+const createElement = (typeTag, classList) => {
+  const element = document.createElement(typeTag);
+  element.classList.add(classList);
+  return element;
+};
+
+export { createElement };
```
Comment (camigomezdev) [inline src/js/helpers/helpers.js:7]: Nice

File: src/index.html [hunk 15]
```diff
@@ -1,14 +1,17 @@
...
-  <link rel="stylesheet" href="./scss/main.scss">
+  <link rel="stylesheet" href="./scss/main.scss" />
 </head>
+
 <body>
   <div id="root"></div>
 
   <script src="./js/index.js"></script>
 </body>
+
```
Comment (camigomezdev) [inline src/index.html]: Please don't send changes in this file

**Review by camigomezdev — APPROVED** [review]

File: src/about.html (hunk 13 above)
Comment (juan-ortizc) [reply src/about.html]: resolved

File: src/js/helpers/helpers.js (hunk 14 above)
Comment (juan-ortizc) [reply src/js/helpers/helpers.js:7]: thank you

File: src/index.html (hunk 15 above)
Comment (juan-ortizc) [reply src/index.html]: resolved
